vtr_flow/misc/yosys/synthesis.tcl

```diff
@@ -66,7 +73,23 @@ techmap -map +/parmys/aldffe2dff.v
 
 opt -full
 
-parmys -a QQQ -nopass -c CCC YYY
+#parmys -a QQQ -nopass -c CCC YYY
+#parmys -a QQQ -c CCC YYY
+#memory -nomap
+#flatten
+
+
+# Separate opt for Parmys execution(verilog or system-verilog)
+if {$env(PARSER) == "default"} {
+    puts "Running Parmys with disables additional passes "
```

Comment:
I'm not sure what this output means.  Do you mean to say "Running parmys with no additional passes." ?